src/main/java/de/zalando/aruha/nakadi/domain/EventType.java

```diff
@@ -49,6 +51,10 @@
     @Nullable
     private EventTypeStatistics defaultStatistic;
 
+    @Valid
+    @Nullable
+    private EventTypeOptions options;
```

Comment:
Ok. But isn't an answer :) I don't understand what should belong to options and what shouldn't.